recipe/meta.yaml

```diff
@@ -11,7 +11,9 @@ source:
 
 build:
   skip: True  # [not linux]
-  number: 0
+  number: 1
+  run_exports:
+    - {{ pin_subpackage('libpciaccess', max_pin='x.x') }}
```

Comment:
This is still black-magic to me but with some convos with @jakirkham it looks like we should choose the max_pin based on how ABI compatible the library has been in the past.

To check the history for libpciaccess you can go to 
https://abi-laboratory.pro/index.php?view=timeline&l=libpciaccess

For more details from the Anaconda folks see https://conda.io/docs/user-guide/tasks/build-packages/variants.html#build-variants

Reply:
Thanks for teaching me about abi-laboratory.pro. 

They seem to be pretty backward compatible, but I'm generally worried about pre 1.x releases. 

They just introduced a breaking change in the 0.14 version.

Maybe we should keep this one as `x.x` and look at the other libraries more carefully?

Reply:
LGTM 👍